### PR TITLE
RU info in explain analyze

### PR DESCRIPTION
**Pull request by songrijie**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

RU was included in `explain analyze` since v7.0.0. This information is missing in user doc. 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-resource-control.md [hunk 1]
```diff
@@ -75,6 +75,10 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
 > - 除了用户执行的查询之外，RU 还可以被后台任务消耗，例如自动统计信息收集。
 > - 上表只列举了本地部署的 TiDB 计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
 
+### 估算 SQL 所消耗的 RU
+
+[explain analyze](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-消耗) 命令可以获取到 SQL 执行时所消耗的 RU 。 需要注意的是，RU 的大小会受缓存的影响 (比如[下推计算结果缓存](/coprocessor-cache.md))，多次执行同一条 SQL 所消耗的 RU 可能会有不同， 因此这个值并不代表每次执行的精确值， 但可以作为估算的参考。
```
Comment (songrijie) [inline tidb-resource-control.md:80]:
```suggestion
[explain analyze](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-消耗) 命令可以获取到 SQL 执行时所消耗的 RU 。需要注意的是，RU 的大小会受缓存的影响 (比如[下推计算结果缓存](/coprocessor-cache.md))，多次执行同一条 SQL 所消耗的 RU 可能会有不同，因此这个值并不代表每次执行的精确值，但可以作为估算的参考。
```

**Review by CabinfeverB — APPROVED** [review]

File: sql-statements/sql-statement-explain-analyze.md [hunk 2]
```diff
@@ -282,6 +282,18 @@ commit_txn: {prewrite:48.564544ms, wait_prewrite_binlog:47.821579, get_commit_ts
 - `write_keys`：事务中写入 `key` 的数量。
 - `write_byte`：事务中写入 `key-value` 的总字节数量，单位是 byte。
 
+### RU (Request Unit) 消耗
+
+[Request Unit (RU)](/tidb-resource-control#什么是-request-unit-ru) 是资源管控对系统资源统一抽象的计量单位。执行计划顶层算子的 `execution info` 会显示 SQL 整体的 RU 消耗。
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain-analyze.md:287]:
```suggestion
[Request Unit (RU)](/tidb-resource-control.md#什么是-request-unit-ru) 是资源管控对系统资源统一抽象的计量单位。执行计划顶层算子的 `execution info` 会显示 SQL 整体的 RU 消耗。
```

File: sql-statements/sql-statement-explain-analyze.md [hunk 3]
```diff
@@ -282,6 +282,18 @@ commit_txn: {prewrite:48.564544ms, wait_prewrite_binlog:47.821579, get_commit_ts
...
+
+```
+RU:273.842670
+```
+
+> **注意：**
+>
+> 这个值代表本次执行的实际 RU 消耗。受缓存的影响 (比如[下推计算结果缓存](/coprocessor-cache))，相同 SQL 每次执行消耗的 RU 可能会有不同。
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain-analyze.md:295]:
```suggestion
> 这个值代表本次执行的实际 RU 消耗。受缓存的影响（比如[下推计算结果缓存](/coprocessor-cache.md)），相同 SQL 每次执行消耗的 RU 可能会有不同。
```

File: tidb-resource-control.md [hunk 4]
```diff
@@ -75,6 +75,10 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
 > - 除了用户执行的查询之外，RU 还可以被后台任务消耗，例如自动统计信息收集。
 > - 上表只列举了本地部署的 TiDB 计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
 
+### 估算 SQL 所消耗的 RU
+
+[explain analyze](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-消耗) 命令可以获取到 SQL 执行时所消耗的 RU 。需要注意的是，RU 的大小会受缓存的影响 (比如[下推计算结果缓存](/coprocessor-cache.md))，多次执行同一条 SQL 所消耗的 RU 可能会有不同，因此这个值并不代表每次执行的精确值，但可以作为估算的参考。
```
Comment (hfxsd) [inline tidb-resource-control.md:80]:
```suggestion
[explain analyze](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-消耗) 命令可以获取到 SQL 执行时所消耗的 RU。需要注意的是，RU 的大小会受缓存的影响（比如[下推计算结果缓存](/coprocessor-cache.md)），多次执行同一条 SQL 所消耗的 RU 可能会有不同，因此这个值并不代表每次执行的精确值，但可以作为估算的参考。
```

File: tidb-resource-control.md [hunk 5]
```diff
@@ -75,6 +75,10 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
 > - 除了用户执行的查询之外，RU 还可以被后台任务消耗，例如自动统计信息收集。
 > - 上表只列举了本地部署的 TiDB 计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
 
+### 估算 SQL 所消耗的 RU
```
Comment (hfxsd) [inline tidb-resource-control.md:78]:
```suggestion
## 估算 SQL 所消耗的 RU
```

File: sql-statements/sql-statement-explain-analyze.md [hunk 6]
```diff
@@ -282,6 +282,18 @@ commit_txn: {prewrite:48.564544ms, wait_prewrite_binlog:47.821579, get_commit_ts
...
+
+[Request Unit (RU)](/tidb-resource-control.md#什么是-request-unit-ru) 是资源管控对系统资源统一抽象的计量单位。执行计划顶层算子的 `execution info` 会显示 SQL 整体的 RU 消耗。
+
+```
+RU:273.842670
+```
+
+> **注意：**
+>
+> 该值仅表示本次执行的实际 RU 消耗。由于受缓存的影响（比如[下推计算结果缓存](/coprocessor-cache.md)），相同 SQL 在每次执行时消耗的 RU 可能会不同。
```
Comment (hfxsd) [inline sql-statements/sql-statement-explain-analyze.md:295]:
```suggestion
> 该值仅表示本次执行的实际 RU 消耗。由于受缓存的影响（比如[下推计算结果缓存](/coprocessor-cache.md)），同一个 SQL 在每次执行时消耗的 RU 可能会不同。
```

**Review by hfxsd — APPROVED** [review]

File: tidb-resource-control.md [hunk 7]
```diff
@@ -75,6 +75,10 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
 > - 除了用户执行的查询之外，RU 还可以被后台任务消耗，例如自动统计信息收集。
 > - 上表只列举了本地部署的 TiDB 计算 RU 时涉及的相关资源，其中不包括网络和存储部分。TiDB Serverless 的 RU 可参考 [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/)。
 
+## 估算 SQL 所消耗的 RU
+
+执行 [explain analyze](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-消耗) 命令可以获取到 SQL 执行时所消耗的 RU。注意 RU 的大小会受缓存的影响（比如[下推计算结果缓存](/coprocessor-cache.md)），多次执行同一条 SQL 所消耗的 RU 可能会有不同。因此这个 RU 值并不代表每次执行的精确值，但可以作为估算的参考。
```
Comment (qiancai) [inline tidb-resource-control.md:80]:
```suggestion
你可以通过 [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-消耗) 语句获取到 SQL 执行时所消耗的 RU。注意 RU 的大小会受缓存的影响（比如[下推计算结果缓存](/coprocessor-cache.md)），多次执行同一条 SQL 所消耗的 RU 可能会有不同。因此这个 RU 值并不代表每次执行的精确值，但可以作为估算的参考。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**